scripts/e2e.sh

```diff
@@ -9,3 +9,5 @@ if [ -n "$1" ]; then
 else
   jest --maxWorkers=1 ./build/e2e/schematics
 fi
+
+rm -rf tmp
```

Comment:
For some reason, `prettier` really wanted to check in here... regardless, the directory does not need to persist after `e2e` tests complete

Reply:
We should fix it differently. It is useful to look at the tmp dir after tests fail. To do that the tmp should be preserved.